sonaranalyzer-dotnet/its/sources/Net5/global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "5.0.100-preview.8.20417.9"
+    "version": "5.0.100-preview.7.20366.6"
```

Comment:
Shouldn't we use `5.0.100-preview.6.20318.15` instead? To be aligned with what is currently installed on the machine?
https://github.com/SonarSource/peachee-languages/blob/packer/windows/scripts/Install-DotNetSdk.ps1#L18

Preview 7 works, but we don't understand why it works. That's risky.
I hope that what is in the script is really installed.

Reply:
I would not take a step back without a strong reason. We use preview 7 for some time now without any problems and I would rather use that version also on the peachee jobs.